presto-hive-hadoop2/pom.xml

```diff
@@ -67,39 +67,39 @@
 
         <!-- for testing -->
```

Comment:
This was before testng previoulsy

Reply:
I guess because testng was the first test-scoped dependency. I understand this comment means "here starts test scoped dependencies"